src/local_scheduler/local_scheduler.cc

```diff
@@ -728,9 +831,15 @@ void process_message(event_loop *loop,
   case MessageType_GetTask: {
```

Comment:
We're not returning resources properly for actor workers that get blocked. The code doesn't appear in this PR, but see the TODOs in the handlers for `MessageType_ReconstructObject` and `MessageType_NotifyUnblocked`. We only call `handle_worker_blocked`/`handle_worker_unblocked` for non-actor workers at the moment.